From e8dcb81bef146e1eb83f6bd9b171719eeb62016f Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Thu, 23 May 2019 18:14:00 +0100
Subject: [PATCH] TOOLS-2261 mountain-gorilla's launch-build script should set
 CONFIGURE_BRANCHES Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 tools/launch-build | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/tools/launch-build b/tools/launch-build
index adf5d9d..7ae7380 100755
--- a/tools/launch-build
+++ b/tools/launch-build
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -113,6 +113,11 @@ case $PROJECT in
 			BUILD_PARAM="${BUILD_PARAM},$(printf '{"name": "PLATFORM_BUILD_FLAVOR", "value": "%s"}' "${PLAT_FLAVOR}")"
 		fi
 		;;
+	"headnode"|"headnode-debug"|"headnode-joyent")
+		if [[ -n "$BRANCH" ]]; then
+			BUILD_PARAM="${BUILD_PARAM},$(printf '{"name": "CONFIGURE_BRANCHES", "value": "bits-branch: %s"}' "${BRANCH}")"
+		fi
+		;;
 esac
 
 if [[ -n $VERBOSE ]]; then
-- 
2.21.0

